sources/packages/forms/src/form-definitions/educationprogramoffering.json

```diff
@@ -902,6 +902,248 @@
+              "label": "What is the course load for this offering?",
+              "labelPosition": "top",
+              "labelWidth": "",
+              "labelMargin": "",
+              "placeholder": "",
+              "description": "",
+              "tooltip": "",
+              "prefix": "",
+              "suffix": "",
+              "widget": {
+                "type": "input"
+              },
+              "displayMask": "",
+              "customClass": "",
+              "tabindex": "",
+              "autocomplete": "",
+              "hidden": false,
+              "hideLabel": false,
+              "mask": false,
+              "autofocus": false,
+              "disabled": false,
+              "tableView": false,
+              "modalEdit": false,
+              "multiple": false,
+              "persistent": true,
+              "delimiter": false,
+              "decimalLimit": "",
+              "requireDecimal": false,
+              "inputFormat": "plain",
+              "protected": false,
+              "dbIndex": false,
+              "truncateMultipleSpaces": false,
+              "encrypted": false,
+              "redrawOn": "",
+              "clearOnHide": true,
+              "customDefaultValue": "",
+              "calculateValue": "",
+              "calculateServer": false,
+              "allowCalculateOverride": false,
+              "validateOn": "change",
+              "validate": {
+                "required": true,
+                "min": 1,
+                "max": 100,
+                "customMessage": "",
+                "custom": "",
+                "customPrivate": false,
+                "json": "",
+                "strictDateValidation": false,
+                "multiple": false,
+                "unique": false,
+                "step": "any",
+                "integer": ""
+              },
+              "errorLabel": "",
+              "errors": "",
+              "key": "courseLoad",
+              "tags": [],
+              "properties": {},
+              "conditional": {
+                "show": null,
+                "when": null,
+                "eq": "",
+                "json": ""
+              },
+              "customConditional": "",
+              "logic": [],
+              "attributes": {},
+              "overlay": {
+                "style": "",
+                "page": "",
+                "left": "",
+                "top": "",
+                "width": "",
+                "height": ""
+              },
+              "type": "number",
+              "input": true,
+              "unique": false,
+              "refreshOn": "",
+              "dataGridLabel": false,
+              "showCharCount": false,
+              "showWordCount": false,
+              "allowMultipleMasks": false,
+              "addons": [],
+              "id": "eybnlcp",
+              "defaultValue": ""
+            },
+            {
+              "label": "invalidCourseLoad",

```

Comment:
Why do we have a separate hidden component to identify invalid course load? 
![image](https://user-images.githubusercontent.com/54600590/170386449-b6370f27-97e7-4acb-89e3-fa944a831552.png)

Reply:
To flag the offering pending, in the below hidden component section, just followed the way they are doing now